src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -201,108 +153,41 @@ export const BaseComment = ({
     return (
         // TODO: Change className to bcs-Comment once FF is removed
         <div className="bcs-BaseComment">
-            <Media
+            <div
                 className={classNames('bcs-Comment-media', {
                     'bcs-is-pending': isPending || error,
                 })}
             >
-                <Media.Figure>
-                    <Avatar
+                <div className="bcs-BaseComment-header">
```

Comment:
Ultimately, this section came to have three top-level components ("metadata", menu, and status). As such, I think it'd probably be most ideal to break it out into a separate component. That said, lots of threaded replies work is lined up for the new sprint which starts today. In order to avoid those tasks from having to fork my fork, we're hoping to limit the scope for now.

Reply:
I agree. Let's limit the scope to what you have now. The biggest priority for us relative to the Boxer release are the UI updates.